assets/js/modules/analytics/components/dashboard/DashboardAllTrafficWidget/TotalUserCount.js

```diff
@@ -128,36 +134,50 @@ export default function TotalUserCount( props ) {
 					</Fragment>
 				) }
 			</h3>
-			{ !! current?.values?.[ 0 ] && (
-				<div className="googlesitekit-data-block__datapoint">
-					{ numFmt( current?.values?.[ 0 ] ) }
-				</div>
+
+			{ gatheringData && zeroDataStatesEnabled && (
+				<GatheringDataNotice style={ NOTICE_STYLE.LARGE } />
+			) }
+
+			{ ( ! gatheringData || ! zeroDataStatesEnabled ) && (
```

Comment:
I think it would be easier to read this as the inverse of the condition above if it were written as follows:
```suggestion
			{ ! ( gatheringData && zeroDataStatesEnabled ) && (
```

Reply:
Sounds good, both works for me :)